tests/example_files/3x3x3.grdecl

```diff
@@ -0,0 +1,54 @@
+SPECGRID
```

Comment:
Is this file still a draft? We can host it on https://github.com/pyvista/vtk-data. And as a follow-up PR add `download_grid_eclipse` or something like that. And use the dataset in the reader docstring examples section

Reply:
This sample file along with the "include" has been modified to cover most of the code lines added. I am okay with hosting it in VTK-data. Not sure how we will handle the include file, though. However, I would rather host an actual geological model that could be shown in the documentation (e.g., maybe a model from here: https://github.com/BinWang0213/PyGRDECL).

Reply:
Ah I see, if its only for testing then we don't need to host it there. It would probably need to be for a follow-up PR anyway since we'd need the reader to load it.